cli/flags.rs

```diff
@@ -152,6 +157,7 @@ pub fn set_flags(
   opts.optflag("", "allow-net", "Allow network access");
   opts.optflag("", "allow-env", "Allow environment access");
   opts.optflag("", "allow-run", "Allow running subprocesses");
+  opts.optflag("", "allow-high-precision", "Allow high precision timestamp");
```

Comment:
I understand the need, but I'm really not so keen to add a new flag for this... I also don't have a good alternative...

Reply:
So do we allow this last new flag?